client/src/menu/character.zig

```diff
@@ -47,27 +49,27 @@ fn emptyCharacter(gameState: *GameState) !void {
             currentTextPosition.y += textSize.y + fieldPadding;
         } else if (std.mem.eql(u8, field.name, "name")) {
             const nameBoxPosition: rl.Vector2 = .{
-                .x = 50,
+                .x = 150,
                 .y = 50,
             };
             const nameLabelPositionY =
                 nameBoxPosition.y - config.buttonFontSize - 2 * config.menuButtonsPadding;
 
             rl.drawText(
                 "Name:",
-                @intFromFloat(nameBoxPosition.x),
+                50,
                 @intFromFloat(nameLabelPositionY),
                 config.buttonFontSize,
                 rl.Color.white,
             );
             const nameText = text{
-                .content = gameState.menu.character_name,
-                .position = &gameState.test_value,
+                .content = gameState.menu.character.create.name,
+                .position = &gameState.menu.character.create.name_position,
             };
             nameText.at(nameBoxPosition);
-            gameState.character.stats.name = gameState.menu.character_name;
+            gameState.world.character.stats.name = gameState.menu.character.create.name;
         } else {
-            std.debug.print("Not editable: .{s}\n", .{field.name});
+            // std.debug.print("Not editable: .{s}\n", .{field.name});
```

Comment:
If we're commenting this line, may as well remove it and delete the empty `else`.